tests/puppeteer/.docker/prestashop/docker_run.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/sh
 
+if [ ! -z "$PS_VERSION" ]; then
+  echo "git checkout $PS_VERSION"
```

Comment:
You can add `set -x` as the beginning of the script if you want to print a trace of simple commands

Reply:
Rename this file in `docker_run_git.sh`
And at the end of this file, run `/tmp/docker_run.sh`.
At least, if the file change in the prestashop/prestashop-git` image, you don't have to edit this file.

Reply:
there is already a file named `docker_run_git.sh` (imported by the image `prestashop-git` and where we run Composer install and call `docker_run.sh` after)

Reply:
Oh, didn't notice that. Thanks =)